docs/labwc-config.5.scd

```diff
@@ -128,6 +129,13 @@ this is for compatibility with Openbox.
 	*fullscreen* enables adaptive sync whenever a window is in fullscreen
 	mode.
 
+*<core><allowTearing>* [yes|no]
+	Allow tearing to reduce input lag. Default is no.
+	This option requires setting the environment variable WLR_DRM_NO_ATOMIC=1.
+
+	*yes* allow tearing if the active window requests it,
+	or its enabled with action ToggleTearing.
```

Comment:
"its" should be "it's", but I'd suggest just "allow tearing if requested by the active window". This action just makes the active window request tearing allowance if its application has not done so, and is covered by this single clause.